sale_pricelist_triple_discount/model/sale.py

```diff
@@ -0,0 +1,51 @@
+#  Copyright (c) 2019 Simone Rubino - Agile Business Group
+#  Copyright (c) 2021 Andrea Cometa - Apulia Software s.r.l.
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    """
```

Comment:
@scigghia Could you remove unused comments ?

Reply:
sure
thanks